front-end/src/app/shared/models/transaction-types/LOAN_REPAYMENT_RECEIVED.model.ts

```diff
@@ -4,14 +4,20 @@ import { SchATransactionType } from '../scha-transaction-type.model';
 import { SchATransaction, ScheduleATransactionTypeLabels, ScheduleATransactionTypes } from '../scha-transaction.model';
 import { STANDARD_CONTROLS, TransactionNavigationControls } from '../transaction-navigation-controls.model';
 import { AggregationGroups } from '../transaction.model';
-import { COMMITTEE, ORGANIZATION_FORM_FIELDS } from 'app/shared/utils/transaction-type-properties';
+import {
+  COMMITTEE,
+  COMMITTEE_FORM_FIELDS,
+  ORGANIZATION_FORM_FIELDS,
+} from 'app/shared/utils/transaction-type-properties';
+import { TemplateMapKeyType } from '../transaction-type.model';
 
 export class LOAN_REPAYMENT_RECEIVED extends SchATransactionType {
   formFields = ORGANIZATION_FORM_FIELDS;
   contactTypeOptions = COMMITTEE;
   title = LabelUtils.get(ScheduleATransactionTypeLabels, ScheduleATransactionTypes.LOAN_REPAYMENT_RECEIVED);
   schema = schema;
   override navigationControls: TransactionNavigationControls = STANDARD_CONTROLS;
+  override inheritedFields = [...COMMITTEE_FORM_FIELDS] as TemplateMapKeyType[];
```

Comment:
Ah this is a failure on my naming.  COMMITTEE_FORM_FIELDS has COMMON_FIELDS and AGGREGATE_FIELDS.  It will inherit the date and amount which we don't want.  I think you'll need to mix COM_FIELDS and ADDRESS_FIELDS.
`[...COM_FIELDS, ...ADDRESS_FIELDS]`

If you can think of a better name to clarify the typical committee based form (currently COMMITTEE_FORM_FIELDS) and JUST the committee specific fields (COM_FIELDS) we could make that improvement here

Reply:
Good catch. I didn't notice it was catching those extra fields. The names are a little confusing when you first come across them, but after working with them for a little while now, they make more sense, so I couldn't think of anything better for naming.